### DISCUSSION TITLE
How do i create a language pack for pt-BR(brazilian portuguese)?

### DISCUSSION THREAD

**Discussion by lucasrochaa** [dlutton/flutter_tts#204] [General]
How do i create a language pack for pt-BR(brazilian portuguese)?

**Comment by dlutton** [comment]
Unfortunately there isn't a way to create language packs for this package.  This package uses the native TTS engines on the device in use.  Apple uses the speech synthesis API and Google uses their Google TTS engine.  Language support depends on the device and what those APIs support.
